src/compiler/scala/tools/nsc/settings/ScalaSettings.scala

```diff
@@ -147,7 +147,7 @@ trait ScalaSettings extends AbsScalaSettings
 
   val YparallelPhases    = PhasesSetting       ("-Yparallel-phases", "Which phases to run in parallel")
   val YparallelThreads   = IntSetting          ("-Yparallel-threads", "Worker threads for parallel compilation", 4, Some((0,64)), _ => None )
-  val YparallelSequential= BooleanSetting      ("-Yparallel-sequential", "Keeps processing units in different threads but do it sequentially")
+  val YparallelSequential = BooleanSetting      ("-Yparallel-sequential", "Keeps processing units in different threads but do it sequentially")
```

Comment:
Now it's not alignment with the rest :)

Reply:
I hate this style of formatting but what can I do... reverted